AssetMappr/presentation/submit_new_asset.py

```diff
@@ -2,27 +2,95 @@
 File: submit_new_asset.py
 Author: Mihir Bhaskar
 
-Desc: This file creates the 'submitting new asset info' component of the app
-Input: 
-   
-
+Desc: This file creates the layout of the 'submit new asset' feature of the app
+Input: A list with the master categories, initialised in app.py
 Output: 
-    - A function that returns an HTML Div containing the user new asset upload features
-    
+    - A function that returns an HTML Div containing the new asset upload popup feature
+
+Other notes:
+    - This interacts with the submit_new_asset_cb in 'application'
+
 """
 import dash
 from dash.dependencies import Input, Output, State
 from dash import dash_table
 from dash import dcc
 from dash import html
+import dash_bootstrap_components as dbc
+import dash_leaflet as dl
 
-def submit_new_asset():
+def submit_new_asset(master_categories):
     
-    return html.Div(
-        id="submit-new-asset",
-        children=[
-            # Entering name
-            html.P("Enter the asset's name"),
-            dcc.Input(id='asset-name', value='', type='text')
-        ]
-    )
\ No newline at end of file
+    return html.Div([
```

Comment:
Generally the section below could do with some clarifying comments- lotta things to keep track of.